src/resmom_win/topology.c

```diff
@@ -110,4 +110,6 @@ int count_gpus() {
  * @return	no. of xeon phi's
  *
  */
-int count_mics() { return 0; }
+int count_mics() { 
```

Comment:
Per the style guide, the return type and function name should be on separate lines. Please explicitly use void if the function takes no arguments.